libs/utils/utils.js

```diff
@@ -833,6 +833,16 @@ async function decorateHeader() {
   }
 }
 
+function decorateAnchors(area) {
+  const anchors = area.querySelectorAll('span[class*="icon-anchor-"]');
+  if (anchors.length === 0) return;
+  anchors.forEach((anchor) => {
+    anchor.id = anchor.classList[1].replace('icon-anchor-', '');
+    anchor.classList.replace(anchor.classList[0], 'anchor');
+    anchor.classList.replace(anchor.classList[1], `anchor-${anchor.id}`);
```

Comment:
Is this class really useful, since we have the ID? Could we not remove it entirely?

Reply:
We might need some CSS changes and I thought it would be easier if the class is there. Also, it follows the exact same track as "icon icon-something" defined by EDS, so wanted to be consistent.

Reply:
Specific styles for each anchor? I can't really think of a use-case for that. Again, this is more of a "this is utils, let's only keep what we need right now" comment.

Reply:
ok, will remove